src/assignments/6-array.js

```diff
@@ -21,3 +31,6 @@ min=1
 
 */
 // start here writing the code for program2
+
+console.log("min=", Math.min(...num));
+console.log("max=", Math.max(...num));
```

Comment:
here, we have to write the program to get the min and the max of the array, you can try to run the for loop.
You can get the logic from here:
(https://www.w3resource.com/c-programming-exercises/array/c-array-exercise-9.php)